Header header
int16 ymin
int16 ymax
int16 xcenter

#Debug stuff
bool debug_included
int16 head_raw
int16 feet_raw
int16 ycenter